optax/losses/_classification.py

```diff
@@ -551,3 +552,64 @@ def sigmoid_focal_loss(
                       loss)
 
   return loss
+
+def ntxent(
+    embeddings: chex.Array,
+    labels: chex.Array,
+    temperature: float = 0.07
```

Comment:
Type hint of temperature can be `chex.Numeric` rather than `float` (maybe people would want to inject their own temperature via a jnp.Array and we don't want to end up having some typing compilation errors).

Also the return type may also be a `chex.Numeric` (correct me if I'm wrong on that one).

Reply:
I tried changing the typehints for the function and ran into an interesting PR error that I am not quite sure how to deal with:
```
optax/_src/clipping.py:153:16: E1102: jnp.greater is not callable (not-callable)
optax/_src/clipping.py:235:17: E1102: jnp.greater is not callable (not-callable)
```
When reverting the changes, the same thing happened? Any advice would be much appreciated!